OloEngine/src/OloEngine/Networking/Prediction/ClientPrediction.cpp

```
@@ -0,0 +1,135 @@
+                    }
+                }
+            }
+        }
+    }
+
+    void ClientPrediction::SetInputApplyCallback(InputApplyCallback callback)
+    {
+        m_ApplyCallback = std::move(callback);
+    }
+
+    void ClientPrediction::SetSmoothingRate(f32 rate)
```

Cppcheck (2.20.0)

[style] The function 'SetSmoothingRate' is never used.

(unusedFunction)